stdlib/os/__init__.pyi

```diff
@@ -621,6 +628,14 @@ if sys.platform != "win32":
         def posix_fadvise(fd: int, offset: int, length: int, advice: int) -> None: ...
     def pread(__fd: int, __length: int, __offset: int) -> bytes: ...
     def pwrite(__fd: int, __buffer: bytes, __offset: int) -> int: ...
+    if sys.platform != "darwin" and sys.version_info >= (3, 7):
+        def preadv(__fd: int, __buffers, __offset: int, __flags: int = ...) -> int: ...
+        def pwritev(__fd: int, __buffers, __offset: int, __flags: int = ...) -> int: ...
```

Comment:
The `buffers` argument for these two is [described in the documentation](https://docs.python.org/3/library/os.html#os.pwritev) as taking a "bytes-like object". What's the standard typeshed annotation for this kind of argument?

Reply:
The docs link to this glossary definition: https://docs.python.org/3/glossary.html#term-bytes-like-object

Reply:
Use `bytes`. It will also include things like `bytearray`, similarly to how a `float` argument also accepts `int`.